waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs

```diff
@@ -26,7 +26,8 @@ bashCompletion = do
     ["db", cmdPrefix] -> listMatchingCommands cmdPrefix dbSubCommands
     _ -> liftIO . putStrLn $ ""
   where
-    commands = ["new", "version", "waspls", "start", "db", "clean", "uninstall", "build", "telemetry", "deps", "info", "completion", "completion:generate"]
+    -- todo(filip): remove duplication
```

Comment:
Should I tackle this now or just go with the current system?

Reply:
Go with the current system -> we actually have a big PR underway to tackle this (optparse-applicative stuff), but it got a bit stuck on some stuff, so for now we are just going with duplication.